docs/draft/sticker-pack.md

```diff
@@ -0,0 +1,77 @@
+---
+title: IPFS gateway for Sticker Pack
```

Comment:
Should the focus of this spec be more on

1) How stickers are communicated, maintained, uploaded here, or
2) How IPFS is used for querying and posting data?

E.g. do we imagine this evolving into more elaborate sticker markets/images/videos etc, or towards IPFS light node/Swarm/Dagger usage?

Reply:
This spec is about `2. How IPFS is used for querying and posting data`. Added link to Sticker Market spec where I think more detailed information is needed. Covered steps that are not directly connected with IPFS on a high level only.